qulice-pmd/src/test/java/com/qulice/pmd/PMDValidatorTest.java

```diff
@@ -84,6 +85,12 @@
     private static final String BRACKETS = "(%s)";
 
     /**
+     * Pattern using plain JUnit assertions.
+     * @checkstyle LineLength (2 lines)
```

Comment:
@vkuchyn you can move the code slightly and get rid of the `LineLength` exclude:
```
private static final String OLD_ASSERTIONS =
    "Avoid using Plain JUnit assertions";
```